src/PrestaShopBundle/DependencyInjection/Compiler/ModulesDoctrineCompilerPass.php

```diff
@@ -110,7 +110,7 @@ private function createAnnotationMappingDriver($moduleNamespace, $moduleEntityDi
             $driverDefinition->addMethodCall('addExcludePaths', [[$indexFile]]);
         }
 
-        return new DoctrineOrmMappingsPass($driverDefinition, [$moduleNamespace], [], false, [$modulePrefix => $moduleNamespace]);
+        return new DoctrineOrmMappingsPass($driverDefinition, [$moduleNamespace], [], false, []);
```

Comment:
https://github.com/doctrine/orm/issues/8818

Reply:
Hmm, so we can't define namespaces for modules' entities? Does it mean module also need to adapt how they use Doctrine entities? If so this will have to be documented in the BC breaks as well